app/uk/gov/hmrc/nisp/views/nirecordVoluntaryContributions.scala.html

```diff
@@ -83,7 +90,11 @@ <h2 class="heading-medium">@messages("nisp.nirecord.voluntarycontributions.h2tit
         <span class="summary">@messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3")</span>
     </summary>
     <div class="panel-indent">
-        <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1") )</p>
+        <p>@Html(messages("nisp.nirecord.voluntarycontributions.h2title.3.linktitle3.message1"))
+          <a href="@{ApplicationConfig.isleManLink}" tabindex="-1" rel="external" target="_blank" id="nir-external-link" data-journey-click="checkmystatepension:external:iom">
+            @messages("nisp.nirecord.gapsinyourrecord.nationalinsurancecredits.link")
```

Comment:
is this right? this changes the content doesnt it?

Reply:
fixed now